src/controllers/userController.js

```diff
@@ -0,0 +1,69 @@
+const {
+  getUsers,
+  getUserById,
+  create,
+  remove,
+  update,
+} = require('../services/userService');
+
+const getAll = (req, res) => {
+  res.statusCode = 200;
+  res.send(getUsers());
+};
+
+const getUser = (req, res) => {
+  const { id } = req.params;
+
+  const user = getUserById(+id);
+
+  if (!user) {
+    return res.sendStatus(404);
+  }
+
+  res.statusCode = 200;
+  res.send(user);
+};
+
+const createUser = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    return res.sendStatus(400);
+  }
+
+  res.statusCode = 201;
+  res.send(create(name));
+};
+
+const removeUser = (req, res) => {
+  const { id } = req.params;
+  const user = getUserById(+id);
+
+  if (!user) {
+    return res.sendStatus(404);
+  }
+
+  res.statusCode = 204;
+  res.send(remove(user.id));
```

Comment:
The `remove` function is called synchronously. If this function involves database or I/O operations, it should be made asynchronous to prevent blocking the event loop. Additionally, sending a response with `res.send()` after setting a 204 status code is incorrect because 204 means 'No Content'. Use `res.end()` instead.